kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/admin/UserTaskAdminServiceBase.java

```diff
@@ -68,6 +69,8 @@ public UserTaskAdminServiceBase(UserTaskAdminService userTaskAdminService, KieSe
 
     public void addPotentialOwners(String containerId, Number taskId, boolean removeExisting, String payload, String marshallingType) {
         logger.debug("About to unmarshall payload '{}' to map of task inputs", payload);
+        if (context.getContainer(containerId) == null) throw new DeploymentNotFoundException("Could not find container with ID: " + containerId);
```

Comment:
I'd suggest to move this to separate method as it's constantly repeated in many places - like checkContainer(containerId) that can throw DeploymentNotFoundException

and please use the if with brackets 
```
if (context.getContainer(containerId) == null) {
    throw new DeploymentNotFoundException("Could not find container with ID: " + containerId);
}
```